### PR TITLE
Add Norwegian Nynorsk localizations to Sileo

### PR DESCRIPTION
**Pull request by worldlydev**
The Norwegian language has 2 written standards. Bokmål which I PRed in #93 and Nynorsk.

### CODE REVIEW

File: Sileo/AppDelegate.swift [hunk 1]
```diff
@@ -127,7 +127,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UITabBarControllerDelegat
         } else if FileManager.default.fileExists(atPath: "/usr/libexec/substrated") {
             appVer += "-substrate"
         } else if FileManager.default.fileExists(atPath: "/usr/libexec/substituted") {
-            appVer += "-hackyA12"
+            appVer += "-unc0ver"
```
Comment (coolstar) [inline Sileo/AppDelegate.swift:130]: No

**Review by coolstar — CHANGES_REQUESTED** [review]

File: Sileo/AppDelegate.swift (hunk 1 above)
Comment (NotZoeyDev) [reply Sileo/AppDelegate.swift:130]: thishowitis

Comment (coolstar) [reply Sileo/AppDelegate.swift:130]: :xqcdesk:

**Comment by Diatrus** [comment]
Use the crowdin

**Pull request closed without merging**